monitor_helper.rb

```diff
@@ -33,11 +33,12 @@ def get_email_domain_name(email)
 
     # remove any tlds
     tlds.each{|t|
+      domain.downcase!
       domain.slice! t
     }
 
     # return what we know about
-    domain.split('.')[-1]
+    domain.split('.')[-1].downcase
```

Comment:
[suggestion] a slightly more ruby version of this whole method would be something like:

```ruby
domain = email.split('@')[1].downcase

domain.gsub(/\.(?:gov|mil|fed\.us)\z/, "").split('.')[-1]
```